zrml/neo-swaps/src/tests/join.rs

```diff
@@ -208,14 +208,15 @@ fn join_fails_on_insufficient_funds() {
             vec![_1_2, _1_2],
             CENT,
         );
+
         assert_noop!(
             NeoSwaps::join(
                 RuntimeOrigin::signed(ALICE),
                 market_id,
                 _100,
                 vec![u128::MAX, u128::MAX]
             ),
-            orml_tokens::Error::<Runtime>::BalanceTooLow
+            pallet_assets::Error::<Runtime, MarketAssetsInstance>::NoAccount
```

Comment:
Do you know off the top of your head why this raises `NoAccount` but a failed `deploy_pool` raises `BalanceLow`?

Reply:
This raises `NoAccount` because `ALICE`'s account for the assets in question was never created, i.e. she has no balance at all. In the `deploy_pool` case, `ALICE`'s account exists because it was created when she received tokens, but the balance she tries to transfer exceeds her available balance.